test/spec/services/angular-apimock.js

```diff
@@ -92,9 +92,10 @@ describe('Service: apiMock', function () {
 			$httpBackend.expect(defaultExpectMethod, defaultExpectPath).respond(404);
 
 			$http(defaultRequest)
-				.success(function () {
-					fail(); // Todo: How to fail the test if this happens?
-					failCb && failCb();
+				.success(/* istanbul ignore next - it is expecting a failure, so should fail tests if happen */
```

Comment:
Ah ok. [Istanbul's documentation](https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md) doesn't mention that being a common practice for test code (as I mentioned in another comment). I'd rather not include the test files for now, so this change can be reverted.